server/itx/server.go

```diff
@@ -231,9 +232,18 @@ func StartServer(ctx context.Context, svr *Server, probeSvr *probe.Server, cfg c
 			log.L().Panic("Failed to stop server.", zap.Error(err))
 		}
 	}()
+	if cfg.API.ReadyDuration > 0 {
```

Comment:
which apis or functions will be impacted? any of them could be started? e.g., heartbeat

Reply:
The waiting occurs after all lifecycle items (e.g., API, P2P) have already started, except for the heartBeat logging and admin API(for debug).

From an external perspective: 
- P2P is already ready and can receive blocks and actions. 
- The API(e.g. http, grpc, wss) is not ready because the `GET /readiness` returns 503.